src/lib/testUtils/documentTestUtils.tsx

```diff
@@ -258,3 +258,36 @@ export function mockTextElement(shape: Partial<ShapeElement> = {}): Element {
     ...shape,
   };
 }
+
+/**
+ * Mock the fullscreen API {@link https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API}
+ *
+ * Set document.fullscreenElement to null.
+ * Mock document.exitFullscreen() to reset document.fullscreenElement to null.
+ * Also mock document.documentElement.requestFullscreen() to set document.fullscreenElement to an empty object.
+ * Both mocked functions dispatch a "fullscreenchange" event on document and return a resolved Promise.
+ */
+export function mockFullscreenApi(): void {
```

Comment:
Taks for me: Have another look if the API can be mocked in a better way.

Reply:
Mocking this API seems to be a strange thing. I've tried everything from `Object.defineProperty(document, 'exitFullscreen'…` to extending the `document` prototype. Neither has worked for me :shrug:  Open for ideas.

Reply:
Let's improve it later on.